packages/components/src/internal/constants.ts

```diff
@@ -248,4 +248,10 @@ export const SM_PIPELINE_JOB_NOTIFICATION_EVENT_START = SM_PIPELINE_JOB_NOTIFICA
 export const SM_PIPELINE_JOB_NOTIFICATION_EVENT_SUCCESS = SM_PIPELINE_JOB_NOTIFICATION_EVENT + '#Success';
 export const SM_PIPELINE_JOB_NOTIFICATION_EVENT_ERROR = SM_PIPELINE_JOB_NOTIFICATION_EVENT + '#Error';
 
+export const LKB_PIPELINE_JOB_NOTIFICATION_EVENT =
```

Comment:
I realize that the LKSM events are in `@labkey/components`, however, it may make more sense to have these events just be declared in LKB itself. There are no references to these events in `@labkey/components` and the only public usage of them is LKB. I would also consider doing the same for LKSM events.